include/yaml-cpp/node/detail/impl.h

```diff
@@ -58,19 +58,24 @@ struct get_idx<Key, typename std::enable_if<std::is_signed<Key>::value>::type> {
 
 template <typename Key, typename Enable = void>
 struct remove_idx {
-  static bool remove(std::vector<node*>&, const Key&) { return false; }
+  static bool remove(std::vector<node*>&, const Key&, std::size_t*) {
```

Comment:
Pass the std::size_t as a reference, not a pointer.

Reply:
That was my preference as well, I've just been on too many projects following the google style guide and I get told to change my references to pointers for output parameters :laughing: